### PR TITLE
Tweaking markets page

### PR DESCRIPTION
**Pull request by IanWoodard**
Makes the entire tab button clickable, makes it more accessible, and adjusts the colors.
image

### CODE REVIEW

File: earn/src/pages/MarketsPage.tsx [hunk 1]
```diff
@@ -30,17 +30,17 @@ import { fetchBorrowerDatas, MarginAccount } from '../data/MarginAccount';
 import { PriceRelayLatestResponse } from '../data/PriceRelayResponse';
 import { getProminentColor } from '../util/Colors';
 
+const UNSELECTED_OPTION_TEXT_COLOR = 'rgba(75, 105, 128, 1)';
```
Comment (haydenshively) [inline earn/src/pages/MarketsPage.tsx:33]: Where is this coming from? If it's from the spec, we should define it in shared, and if not, let's switch to something that is -- will grey 600 or 700 work?

File: earn/src/pages/MarketsPage.tsx [hunk 2]
```diff
@@ -52,7 +52,7 @@ const HeaderSegmentedControlOption = styled(Text).attrs((props: { isActive: bool
     content: ${(props) => (props.isActive ? "''" : 'none')};
     background-color: white;
 
-    top: 40px;
+    top: 100%;
```
Comment (haydenshively) [inline earn/src/pages/MarketsPage.tsx:55]: nifty

File: earn/src/pages/MarketsPage.tsx [hunk 3]
```diff
@@ -358,23 +358,27 @@ export default function MarketsPage() {
...
-          </HeaderSegmentedControlOption>
-          <HeaderSegmentedControlOption
-            size='M'
-            isActive={selectedHeaderOption === HeaderOptions.Borrow}
-            onClick={() => setSelectedHeaderOption(HeaderOptions.Borrow)}
-          >
-            Borrow
-          </HeaderSegmentedControlOption>
+        <div>
+          <div className='flex flex-row' role='tablist'>
```
Comment (haydenshively) [inline earn/src/pages/MarketsPage.tsx:362]: 👍

**Review by haydenshively — APPROVED** [review]

File: earn/src/pages/MarketsPage.tsx (hunk 1 above)
Comment (IanWoodard) [reply earn/src/pages/MarketsPage.tsx:33]: It is the same color used for the navbar

Comment (IanWoodard) [reply earn/src/pages/MarketsPage.tsx:33]: Just checked, this is GREY_400. Will add it to shared

**Pull request merged**